app/src/admin/java/com/mailerdaemon/app/admin/AdminActivity.kt

```diff
@@ -1,12 +1,137 @@
 package com.mailerdaemon.app.admin
 
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.database.Cursor
+import android.net.Uri
 import android.os.Bundle
+import android.provider.MediaStore
+import android.util.Log
+import android.view.View
+import android.widget.ArrayAdapter
+import android.widget.Spinner
 import androidx.appcompat.app.AppCompatActivity
-import com.mailerdaemon.app.R
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import com.mailerdaemon.app.*
+import com.mailerdaemon.app.utils.ImageUploadCallBack
+import com.mailerdaemon.app.utils.UploadData
+import kotlinx.android.synthetic.admin.activity_admin.*
+
+
+class AdminActivity : AppCompatActivity(), ImageUploadCallBack {
+    private var path: String? = null
+    private var downloadUrl: String? = null
+    private val topics = arrayOf("event", "campus", "placement")
+    private val mTitle = mapOf("event" to "Event", "placement" to "PlacementDeamon", "campus" to "CampusDeamon")
+    private val mClickAction = mapOf("event" to "EventActivity", "placement" to "PlacementActivity", "campus" to "CampusActivity")
 
-class AdminActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_admin)
+
+        val spinner = findViewById<Spinner>(R.id.spinner_tag)
+        if (spinner != null) {
+            val adapter = ArrayAdapter(this,
+                android.R.layout.simple_spinner_item, topics)
+            spinner.adapter = adapter
+        }
+
+        bt_img_notification.setOnClickListener {
+            if(ContextCompat.checkSelfPermission(this, android.Manifest.permission.READ_EXTERNAL_STORAGE)==
+                PackageManager.PERMISSION_DENIED)
+            {
+                ActivityCompat.requestPermissions(this, arrayOf(android.Manifest.permission.WRITE_EXTERNAL_STORAGE), 1000)
+            } else {
+              pickImageFromGallery()
```

Comment:
Change here to open the default file manager and check if its result is image and size is less than 10 mb.